suministrospr/suministros/templates/suministros/suministro_detail.html

```diff
@@ -29,7 +29,7 @@
       <div class="flex flex-row text-sm text-gray-600">
         <div>
           Actualizado:
-          {% language 'es' %}{{ object.modified_at|timezone:"America/Puerto_Rico"|date:"j \d\e F \d\e Y \a \l\a\s  P" }}{% endlanguage %}
+          {% language 'es' %}{{ object.modified_at|timezone:"America/Puerto_Rico"|date:"j \d\e F \d\e Y \a \l\a\s  P"|lower }}{% endlanguage %}
```

Comment:
Note to self: 5 years later seems like a good time to revisit a comment on why Django's locale uses Enero instead of enero 😄 

https://code.djangoproject.com/ticket/17426#comment:2

Reply:
![](https://thisiswhatheidithinkshome.files.wordpress.com/2019/02/anigif_enhanced-3572-1534972444-1.gif)